core/store/src/flat_state.rs

```diff
@@ -744,27 +744,27 @@ pub mod store_helper {
         shard_layout: ShardLayout,
         shard_id: u64,
         store: &'a Store,
-        from: Option<Vec<u8>>,
-        to: Option<Vec<u8>>,
+        from: Option<&'a Vec<u8>>,
+        to: Option<&'a Vec<u8>>,
```

Comment:
nit: Any reason why this isn't accepting slices? `&'a [u8]` is more flexible to use on the caller. On the callee they are virtually the same. (In this case you can even remove `.map(|x| x.as_slice())`)